src/client/stories/login-form.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import LoginForm from "../components/login-form";
+import { Button } from "@storybook/react/demo";
+import { withKnobs, text } from "@storybook/addon-knobs";
+
+export function LoginFormStory() {
+  const email = text("Email", "test@test.com");
+  const password = text("password", "test");
+
+  return (
+    <div>
+      <LoginForm Email={email} Password={password} />
+      <Button>Login</Button>
```

Comment:
Why is there a `Button` here? there is already a login button inside the `LoginForm`?

Reply:
resolved